app/editor/src/features/home/InfoText.tsx

```diff
@@ -1,4 +1,20 @@
+import { defaultAlert } from 'features/admin/alerts/constants';
+import React from 'react';
+import { useAlerts } from 'store/hooks/admin';
+import { IAlertModel, Show } from 'tno-core';
+
 export const InfoText: React.FC = () => {
+  const [, api] = useAlerts();
+  const [alert, setAlert] = React.useState<IAlertModel>(defaultAlert);
+
+  React.useEffect(() => {
```

Comment:
You will need a separate controller in the editor area with one endpoint that has no security.